src/components/sidebar/common/SectionResult.js

```diff
@@ -14,12 +15,23 @@ export const styles = {
 
 function SectionResult({ addSection, sectionHover, sectionHoverOff, section, disabled, classes }) {
   const sectionIsUnscheduled = section.schedules.some(isUnscheduled);
+  const { enqueueSnackbar } = useSnackbar();
+
+  const message = 'Successfully added';
+  const handleClick = () => {
+    enqueueSnackbar(message, {
+      variant: 'success',
+    });
+  };
 
   return (
     <ListItem
       key={section.id}
       button
-      onClick={() => addSection(section)}
+      onClick={() => {
+        addSection(section);
+        handleClick();
```

Comment:
Seems like to me the whole function that's provided to `onClick` should be called `handleClick`, and `addSection` can be called from within `handleClick`.

Reply:
I agree with this! The code then becomes a bit more concise:
`onClick={handleClick}`